editor/js/inputs.js

```diff
@@ -39,12 +43,25 @@ function load(inputs) {
   });
 }
 
+/**
+ * Populates a single reaction input section in the form.
```

Comment:
"Creates and populates"? since this method involves a call to `add`, as opposed to loadInputUnnamed which doesn't actually add a new div

Reply:
That's fair; I can update this in many places.

Reply:
Done in 746da3e5377414e24279e939c62e2d70df7fc0ef